### DISCUSSION TITLE
Datagrid Column with column span

### DISCUSSION THREAD

**Discussion by NeoCoderMatrix86** [Megabit/Blazorise#4454] [Q&A]
Hello,

I'm trying to move from a table to datagrid component, but I'm facing a problem: On the current table I have a table row above the row with edit components.

For example:
```
<TableBody>
	<TableRow>
		<TableRowCell ColumnSpan="9">
			[...]
		</TableRowCell>
	</TableRow>
	<TableRow>
		<TableRowCell>
			[...]
		</TableRowCell>
	</TableRow>
</TableBody>
```

Is there any possibility that I can have a datagrid column with column span?

**Comment by stsrki** [comment]
This is not supported at the moment. You can try adding `colspan` on a column and hiding other columns, but it is not guaranteed to work.

**Example**

```razor
<DataGridColumn colspan="9" ...>
```

**Hide other columns**

```razor
<DataGridColumn Field="Name1" Display="Display.None" ...>
<DataGridColumn Field="Name2" Display="Display.None" ...>
<DataGridColumn Field="Name3" Display="Display.None" ...>
```

**Reply by NeoCoderMatrix86** [reply]
Well, in this case how do I get another row?
